simple_rl/agents/func_approx/dsc/SkillChainingAgentClass.py

```diff
@@ -188,7 +189,7 @@ def get_init_q_value_for_new_option(self, newly_trained_option):
 		state_option_pairs = newly_trained_option.final_transitions
 		q_values = []
 		for state, option_idx in state_option_pairs:
-			q_value = global_solver.get_qvalue(np.array(state.features()), option_idx)
+			q_value = global_solver.get_qvalue(state.features(), option_idx)
```

Comment:
Make sure that `state.features()` is still a `np.array`. I think it is a `LazyFrame` or `FrameStack` thing rn